app/config/microserviceWiring.scala

```diff
@@ -33,11 +37,21 @@ trait Hooks extends HttpHooks with HttpAuditing {
 
 object MicroserviceAuditConnector extends AuditConnector with RunMode {
   override lazy val auditingConfig = LoadAuditingConfig(s"auditing")
+
+  override protected def mode: Mode = Play.current.mode
+
+  override protected def runModeConfiguration: Configuration = Play.current.configuration
 }
 
-trait WSHttp extends HttpGet with WSGet with HttpPut with WSPut with HttpPost with WSPost with HttpDelete with WSDelete with Hooks with AppName
+trait WSHttp extends HttpGet with WSGet with HttpPut with WSPut with HttpPost with WSPost with HttpDelete with WSDelete with Hooks with AppName{
+  override protected def actorSystem: ActorSystem = akka.actor.ActorSystem()
```

Comment:
Your current code will create an actorSystem. Please refer to the static dependency of actor system that is in use.
```suggestion
  override protected def actorSystem: ActorSystem = Play.current.actorSystem
```